app/controllers/loading/UnLocodeController.scala

```diff
@@ -68,7 +68,7 @@ class UnLocodeController @Inject() (
         .fold(
           formWithErrors => Future.successful(BadRequest(view(formWithErrors, departureId, mode))),
           value =>
-            service.doesUnLocodeExist(value).flatMap {
+            service.doesUnLocodeExist(value.toUpperCase).flatMap {
```

Comment:
Consider handling this in the form provider with
`"value" -> adaptedText(s"$prefix.error.required")(_.toUpperCase)`